account_invoice_merge_attachment/models/account_invoice.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016-2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
```

Comment:
This model doesn't exist since V14. 

you should inherit from account.move and rename the file.

maybe a module renaming is required also. (account_module_merge_attachment). It depends if it makes sense to use merge attachment feature, for "non invoices" account move....